src/auth/jwtaccess.ts

```diff
@@ -71,12 +74,11 @@ export class JWTAccess {
     // The payload used for signed JWT headers has:
     // iss == sub == <client email>
     // aud == <the authorization uri>
-    const payload = {iss: this.email, sub: this.email, aud: authURI, exp, iat};
-    const assertion = {
-      header: {alg: 'RS256'} as jws.Header,
-      payload,
-      secret: this.key
-    };
+    // NOTE: Users can also override the values for iss / sub / aud / etc
+    // with additionalClaims.
```

Comment:
This worries me somewhat. Are these potentially overridable properties documented anywhere? I am worried about accidental overriding of these properties because the user got the `additionalClaims` object from somewhere else and passed it on. It might lead to spurious errors. We even have strong types here to avoid the trap. `additionalClaims` is simply an `any`.

Reply:
You're right, and this is a really good catch.  I updated the code to validate that users don't pass any of the pre-populated properties, and changed the test to do the opposite.  PTAL!